packages/legacy-events/ResponderEventPlugin.js

```diff
@@ -282,7 +282,7 @@ to return true:wantsResponderID|                            |
                                +                            + */
 
 /**
- * A note about event ordering in the `EventPluginHub`.
+ * A note about event ordering in the `EventPluginRegistry`.
```

Comment:
We inject into the `EventPluginRegistry` now, so the comment changes are to reflect that.